index.js

```diff
@@ -94,9 +94,9 @@ function recurse(parent, theirs, mine, path, options) {
  */
 function processKeyValuePair(key, value, parent, theirs, mine, path, options) {
   options = options || {};
-  var parentValue = typeof parent !== 'undefined' ? parent[key] : undefined;
-  var theirsValue = typeof theirs !== 'undefined' ? theirs[key] : undefined;
-  var mineValue = typeof mine !== 'undefined' ? mine[key] : undefined;
+  var parentValue = typeof parent !== 'undefined' ? (parent !== null ? parent[key] : parent ) : undefined;
```

Comment:
Think this might be better as double clauses on the first conditional instead of nested ternaries. Is there a difference between returning undefined or null?